subprojects/gradle/runner/client/src/main/kotlin/com/avito/runner/scheduler/listener/LogListener.kt

```diff
@@ -30,14 +30,12 @@ internal class LogListener : TestListener {
             is TestCaseRun.Result.Ignored -> "IGNORED"
             is TestCaseRun.Result.Failed.InRun -> "FAILED"
             is TestCaseRun.Result.Failed.InfrastructureError -> {
-                device.logger.warn(result.errorMessage, result.cause)
+                device.logger.warn("${test.testName} LOST", result.error)
```

Comment:
Interesting that we already have a general place where we logging all test transitions